### PR TITLE
🐴 Access Member Profile through a link

### LINKED ISSUE #613: Allow linking to member profile
See #50.

The copy button was added in #602.

We need to allow to link to a member profile. Because this sidebar can be shown from various places the link should always link to the members subsection of the App.

### PR DESCRIPTION
**Pull request by p-sad**
Closes #613

### CODE REVIEW

File: packages/ui/src/memberships/components/MemberProfile/MemberProfile.tsx [hunk 1]
```diff
@@ -67,7 +69,7 @@ export const MemberProfile = React.memo(() => {
                   <EditSymbol />
                 </EditMembershipButton>
               )}
-              <ButtonGhost size="small">
+              <ButtonGhost size="small" onClick={() => copyValue(`${window.location.host}/#/members/${member.id}`)}>
```
Comment (p-sad) [inline packages/ui/src/memberships/components/MemberProfile/MemberProfile.tsx:72]: I wanted to use either this or `window.location.origin`

Comment (jodator) [reply packages/ui/src/memberships/components/MemberProfile/MemberProfile.tsx:72]: I think that in this case it is OK. I don't recall what's best, but host AFAIR is good enough.

Comment (jodator) [reply packages/ui/src/memberships/components/MemberProfile/MemberProfile.tsx:72]: However, it looks like we should have a way to handle links globally. (ATM requires changing links in a couple of places). But that's for later.

**Review by jodator — APPROVED** [review]

File: packages/ui/src/memberships/components/MemberProfile/MemberProfile.tsx (hunk 1 above)
Comment (jodator) [reply packages/ui/src/memberships/components/MemberProfile/MemberProfile.tsx:72]: Moved to: #768.

**Pull request merged**